docs/release/trg-0/trg-8-04.md

```diff
@@ -0,0 +1,167 @@
+---
+title: TRG 8.04 - KIT Structure
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 18-Jan-2024  | Initial contribution                   |
+
+## Project Structure
+
+Following our project structure, the collection of our KITs documentation is developed in the `./docs-kits/kits` folder, where each KIT is a subfolder called by its name for organisation purposes. The `Data Chain KIT`, for example, is defined here: `./docs-kits/kits/Data Chain Kit`.
```

Comment:
I would really appreciate, if we could stop recommending spaces in directory names. We build a website from this source and always have to fight with URL encoding. You can also see that in some of the links provided in this documentation. 

My suggestion would be kebab-case, since this seems to be the one, currently used the most in this repo. 
so `/docs-kits/kits/data-chain-kit` instead of `/docs-kits/kits/Data Chain Kit`